package.json

```diff
@@ -38,7 +38,8 @@
     "redux": "4.0.5",
     "redux-thunk": "2.3.0",
     "reselect": "4.0.0",
-    "scss": "^0.2.4"
+    "scss": "^0.2.4",
```

Comment:
"It's a training, so I'd say let's stick to one approach" (© Anton Vlasik)

Reply:
I'm not using it, I just haven't uninstalled it. I'll do that